src/pages/Calculator/Steps/MixRatios/index.js

```diff
@@ -280,6 +280,9 @@ const MixRatio = ({
 
                 <Grid item xs={12} pt="1rem">
                   <Button
+                    sx={{
+                      borderRadius: '1rem',
+                    }}
                     onClick={() => {
                       setShowSteps({ ...showSteps, [seed.label]: !showSteps[seed.label] });
                     }}
```

Comment:
Can you also change the text on line 291 from `'Change My Rate'` to `'View Calculations'`?